modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/DDMFormViewFormInstanceRecordsManagementToolbarPropsTransformer.js

```diff
@@ -32,16 +32,35 @@ export default function propsTransformer({
 								`${portletNamespace}ddmFormInstanceRecord`
 							);
 
+							const managementBarSelection = document.querySelector(
+								'.management-bar'
+							)?.textContent;
+
 							if (form && searchContainer) {
-								postForm(form, {
-									data: {
-										deleteFormInstanceRecordIds: getCheckedCheckboxes(
-											searchContainer,
-											`${portletNamespace}allRowIds`
-										),
-									},
-									url: deleteFormInstanceRecordURL,
-								});
+								if (
+									managementBarSelection &&
+									managementBarSelection.includes(
+										'All Selected'
```

Comment:
@rijema I don't like relying on the class name `.management-bar` and the text `All Selected` to make this work.

- If the team responsible for the `clay:management-toolbar taglib` changes the class name, this code will no longer work.
- What happens if I display the page in another language? The text will no longer be `All Selected`, right?

Could you check how other components handle this scenario so we can do the same?

Thanks!

Reply:
Thanks for your inputs! It makes a lot of sense!! 

Would you mind if I ask we can consider this approach since the current Management toolbar component is deprecated so, in theory, no changes in the message could be done?  https://clayui.com/docs/components/management-toolbar.html

I can get the respective translation for the label  by getting on the backend using the locale with LanguageUtil. I'm not sure if Liferay.Util.Language would receive the proper translation.

Reply:
I also will check on the Document Library.

Reply:
> I can get the respective translation for the label by getting on the backend using the locale with LanguageUtil. I'm not sure if Liferay.Util.Language would receive the proper translation.

Please update the PR with this change so I can analyze it.

Reply:
Hi @carolmariaabb , done. ✅